core/src/test/java/com/linecorp/armeria/internal/common/ContextAwareFutureTest.java

```diff
@@ -83,8 +83,8 @@ void makeContextAwareCompletableFutureWithDifferentContext(
             assertThat(callbackCalled.called.get()).isFalse();
             verify(appender, atLeast(0)).doAppend(eventCaptor.capture());
             assertThat(eventCaptor.getAllValues()).anySatisfy(event -> {
-                assertThat(event.getLevel()).isEqualTo(Level.WARN);
-                assertThat(event.getMessage()).startsWith("An error occurred while pushing");
+                assertThat(event.getLevel()).isEqualTo(Level.DEBUG);
+                assertThat(event.getMessage()).startsWith("Router created for");
```

Comment:
Used assertions from `Java12ContextAwareFutureTest`. If you prefer, I can also remove these log assertions in all three test classes.